business/src/main/resources/org/mskcc/cbio/portal/persistence/ClinicalFieldMapper.xml

```diff
@@ -12,6 +12,16 @@
     <result property="priority" column="PRIORITY"/>
 </resultMap>
 
+<resultMap id="ClinicalFieldResultWithStats" type="DBClinicalFieldWithStats">
+    <id property="attr_id" column="ATTR_ID"/>
+    <result property="display_name" column="DISPLAY_NAME"/>
+    <result property="description" column="DESCRIPTION"/>
+    <result property="datatype" column="DATATYPE"/>
+    <result property="is_patient_attribute" column="PATIENT_ATTRIBUTE"/>
+    <result property="priority" column="PRIORITY"/>
+    <result property="attribute_count" column="ATTRIBUTE_COUNT"/>
+</resultMap>
```

Comment:
I think this code you can skip in the new api layer. There mapping seems to be done directly in sql query with SQL aliases. E.g. Select DISPLAY_NAME as display_name from ...

Reply:
just fyi